fluentgen/pom.xml

```diff
@@ -33,6 +40,12 @@
       <version>3.15.13</version>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.googlejavaformat</groupId>
+      <artifactId>google-java-format</artifactId>
+      <version>1.8_customized-SNAPSHOT</version>
```

Comment:
A customized google-java-format, tweaked to match our checkstyle.

Reply:
However it requries Java 11.

Reply:
Use java version and reflect to call it. So it will pass compile, and disabled in runtime if version is lower than 11.